mne/preprocessing/artifact_detection.py

```diff
@@ -527,7 +530,7 @@ def annotate_break(raw, events=None,
         onset=break_onsets,
         duration=break_durations,
         description=['BAD_break'],
-        orig_time=raw.info['meas_date']
+        orig_time=raw.info['meas_date'],
```

Comment:
You added the comma on purpose?

(If yes, all good; but just wanted to make sure it's not an accident)

Reply:
I was manually reverting my prior change and left the comma there - I usually leave trailing commas, because then the git diff is clearer the next time around someone adds a new row.

But yes, in this case it's actually unclean :grimacing: